ext/js/pages/settings/profile-controller.js

```diff
@@ -261,6 +261,10 @@ export class ProfileController {
         // Update profile index
         if (settingsProfileIndex === profileIndex) {
```

Comment:
Also what's the difference between `settingsProfileIndex` and `profileIndex`?

Reply:
`settingsProfileIndex` is editing profile index, `profileIndex` is the index to be deleted. Maybe should rename this?